src/pages/transactions/[tid].tsx

```diff
@@ -0,0 +1,735 @@
+                className="text-white-700 w-[101px] tracking-[0.01em]"
+              >
+                {fixedTitle.gasPrice}
+              </div>
+            </div>
+            <div className="flex lg:text-left md:text-left text-right">
+              <NumericFormat
+                data-testid="transaction-gas-price"
+                className="text-white-50 whitespace-normal lg:w-[247px] md:w-[137px] w-[132px] tracking-[0.01em]"
+                thousandSeparator
+                value={gasPrice.value}
+                decimalScale={8}
+                suffix={` ${gasPrice.symbol}`}
+              />
+            </div>
+          </div>
+          <div className="flex flex-row">
+            <div className="flex flex-row items-center grow">
+              <div
+                data-testid="transaction-gas-limit-title"
+                className="text-white-700 w-[67px] tracking-[0.01em]"
+              >
+                {fixedTitle.gasLimit}
+              </div>
+              <Tooltip text={fixedTitle[9]}>
+                <InfoIcon
+                  data-testid="transaction-gas-limit-icon"
+                  className="ml-[9.33px] mr-[8.67px]"
+                />
+              </Tooltip>
+            </div>
+
+            <div className="flex lg:text-left md:text-left text-right">
+              <div className="text-white-50 tracking-[0.01em] whitespace-normal lg:w-[247px] md:w-[137px] w-[132px]">
+                {useUnitSuffix(gasLimit)}
+              </div>
+            </div>
+          </div>
+          <div className="flex flex-row">
+            <div className=" grow">
+              <div
+                data-testid="transaction-gas-used-title"
+                className="text-white-700 w-[101px] tracking-[0.01em]"
+              >
+                {fixedTitle.gasUsed}
+              </div>
+            </div>
+            <div className="flex flex-col">
+              <NumericFormat
+                data-testid="transaction-gas-used"
+                className="text-white-50 tracking-[0.01em] whitespace-normal lg:w-[247px] md:w-[137px] w-[132px] lg:text-left md:text-left text-right"
+                thousandSeparator
+                value={gasUsed}
+                decimalScale={0}
+              />
+              <NumericFormat
+                data-testid="transaction-gas-used-percentage"
+                className="text-white-700 tracking-[0.02em] text-xs mt-1 lg:w-[247px] md:w-[137px] w-[132px] lg:text-left md:text-left text-right"
+                thousandSeparator
+                value={gasUsedPercentage}
+                decimalScale={2}
+                suffix="%"
+              />
+            </div>
+          </div>
+        </div>
+        {type === "tokenized" && (
+          <div className="flex flex-col gap-y-6 lg:w-1/2 md:w-1/2">
+            <div
+              data-testid="transaction-token-transferred-title"
+              className="text-white-50"
```

Comment:
consider refactoring, since it is more than once
```suggestion
              className="text-white-50  font-bold"
```